doc/deployment/README.md

```diff
@@ -49,9 +68,17 @@ First, it's necessary to start some node/peer services:
 docker run \
     --network host \
     emcmongoose/mongoose[-<TYPE>] \
-    --run-node [\
-    --load-step-node-port=<PORT>]
+    --run-node 
 ```
+> NOTE: Mongoose uses `1099` port for RMI between mongoose nodes and `9999` for REST API. If you run several mongoose nodes on the same host (in different docker containers, for example) or if the ports are used by another service, then ports can be redefined:
```

Comment:
Consider adding a simple example with 1 additional node and 1 master node showing usage of different ports

Reply:
https://github.com/emc-mongoose/mongoose-base/blob/BASE-1389-documentation/doc/deployment/README.md#custom-ports